examples/pytorch/question-answering/run_distill_qa.py

```diff
@@ -0,0 +1,791 @@
+#!/usr/bin/env python
```

Comment:
Is there anyway we can combine the distillation code with the regular run_qa script and have it as an option to enable?

Reply:
done